### DISCUSSION TITLE
Is `style props` accepting random `inline styles` documented somehwere?

### DISCUSSION THREAD

**Discussion by coodoo** [chakra-ui/chakra-ui#7045] [General]
While reading the doc I found an [interesting example here](https://chakra-ui.com/docs/hooks/use-token#usage):

```javascript
<Box p={4} boxShadow={`inset 0 4px 0 ${red100}, 0 0 8px ${blue200}`}>
```

Notice the `boxShadow` style prop accepts random inline styles instead of just `xs`, hence opening the door to freely adding custom styles to any props when needed without using `sx` all the time, this is really a nice surprise but wondering is this documented somewhere?

**Comment by wrux** [comment]
The value inside `boxShadow` is a template literal, so it will just resolve to a string. It's also possible to do something like:

```jsx
<Box p={4} __css={{
  boxShadow: `inset 0 4px 0 ${red100}, 0 0 8px ${blue200}`,
}}>
```

**Reply by coodoo** [reply]
Yup, `__css` and `sx` were the way to go for something like this, hence I'm a bit surprised to see that one could simply put template literal as value to the prop, this is very nice, I'm just wondering why this feature isn't documented anywhere for more people to use?

**Reply by wrux** [reply]
Yea right. Might be worth opening a PR to add some documentation
